frontend/lib/tests/form-fields.test.tsx

```diff
@@ -79,40 +71,45 @@ describe('HiddenFormField', () => {
     const defaultProps: BaseFormFieldProps<string> = {
       ...fakeBaseFieldProps({ value: '' }),
     };
-    return shallow(
+    return new ReactTestingLibraryPal(
       <HiddenFormField
         {...defaultProps}
         {...props}
       />
     );
   }
 
+  afterEach(ReactTestingLibraryPal.cleanup);
+
   it('renders name and value attrs', () => {
     for (let value of ['', 'blah']) {
-      const html = makeField({ name: 'boop', value }).html();
+      const html = makeField({ name: 'boop', value }).rr.container.innerHTML;
       expect(html).toContain('name="boop"');
       expect(html).toContain(`value="${value}"`);
     }
   });
 
   it('renders value attr as "on" if it is true', () => {
-    const html = makeField({ name: 'boop', value: true }).html();
+    const html = makeField({ name: 'boop', value: true }).rr.container.innerHTML;
     expect(html).toContain('name="boop"');
     expect(html).toContain('value="on"');
   });
 
-  it('does not render value attr if it is undefined, null, or false', () => {
-    for (let value of [undefined, false, null]) {
-      const html = makeField({ name: 'boop', value }).html();
+  [undefined, false, null].forEach(value => {
+    it(`renders empty value attr if it is ${value}`, () => {
+      const html = makeField({ name: 'boop', value }).rr.container.innerHTML;
       expect(html).toContain('name="boop"');
-      expect(html).not.toContain('value');
-    }
+      expect(html).toContain('value=""');
+    });
   });
```

Comment:
This is the only part of the migration that felt _really_ weird.  I'm not sure why the `value` attribute wasn't showing up in the HTML rendering using enzyme, while it did show up as an empty value when using RTL.  My only guess is that enzyme's `html()` method was perhaps actually serializing HTML using some custom algorithm that introspected a component's props, while RTL was using jsdom, but I'm not sure.